setup.py

```diff
@@ -47,7 +47,8 @@
 
 
 # Make sure we have the right Python version.
-if sys.version_info[:2] < (3, 6):
+PYVER = (3, 6)
+if sys.version_info[:2] < PYVER:
     sys.stderr.write(
         "Biopython requires Python 3.6 or later. "
```

Comment:
The message needs to use ``PYVER`` too.

Reply:
Although see comment below, use and old fashioned percent formatting call so that ``setup.py`` works on old Python.